impl/maven-impl/src/test/java/org/apache/maven/impl/DefaultPluginXmlFactoryTest.java

```diff
@@ -252,19 +251,17 @@ void locateExistingPomWithFilePathShouldReturnSameFileIfRegularFile() throws IOE
     }
 
     @Test
-    @DisabledOnOs(
-            value = WINDOWS,
-            disabledReason = "windows related issue https://github.com/apache/maven/pull/2312#issuecomment-2876291814")
-    void readFromUrlParsesPluginDescriptorCorrectly() throws Exception {
```

Comment:
the builder took (and still takes) an input stream not a URL.  So this is renamed accordingly.
If it is expected that the test was using a Url then the test was already incorrectly implemented.